packages/manager/apps/telecom/src/components/telecom/telephony/group/consumption/pie-chart/telephony-group-consumption-pie-chart.directive.js

```diff
@@ -98,6 +104,10 @@ angular
                 currentArc.classed("clicked", !clicked);
             }); */
 
+        angular.element($window).bind('resize', function() {
```

Comment:
you should unbind when scope is destroyed

Reply:
I have fixe that on last commit, can you re-check please ? :)